projects/zookeeper.cmake

```diff
@@ -23,7 +23,7 @@ set(PRO_ZOOKEEPER
   GIT_ORIGIN ${ZK_REPO}
   GIT_TAG release-${ZK_VER}
   DLURL ${ZK_REPO}/archive/release-${ZK_VER}.tar.gz
-  DLMD5 23df0e0bccf5d05d8190fd8ef459947c
+  DLMD5 790c9b028f2f9c6ed17938a396365b74
```

Comment:
How did the hash change and not the version?

Reply:
@bnhhughes it changed on the server side. They must have posted a new file

Reply:
Hmm, that's weird...